src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -219,6 +244,17 @@ def process_input(conf):
         with open(metafile, "w", encoding="utf8") as json_file:
             json.dump(_process_object_info(obj_info), json_file, indent=2)
 
+        # remove pre-exsit soft links and reset
+        csd_upa_dir = os.path.join(conf.csd, upa)
+        if os.path.isdir(csd_upa_dir):
+            shutil.rmtree(csd_upa_dir)
+        os.makedirs(csd_upa_dir)
+
+        # soft link .fa file and meta file
+        # from sourcedata/WS/<wsid>/<upa> to sourcedata/collection/soure_version/<upa>
+        os.symlink(dst, os.path.join(csd_upa_dir, f"{upa}.fa"))
+        os.symlink(metafile, os.path.join(csd_upa_dir, f"{upa}.meta"))
```

Comment:
Just softlink `csd_upa_dir` to `dstd`, so any other files we add to the source dir will automatically show up

Reply:
Any ideas how to correctly soft link two dirs? I somehow cannot `cd` to dst dir after soft link. 

```
import os
src = "/Users/sean/Desktop/ouput_dir"
dst = "/Users/sean/Desktop/col_dir"
os.symlink(src, dst, target_is_directory=True)
```

Reply:
It seems above code works in Linux env 👍

Reply:
:+1: